core/pref.h

```diff
@@ -96,6 +96,7 @@ struct preferences {
 	dive_computer_prefs_t dive_computer2;
 	dive_computer_prefs_t dive_computer3;
 	dive_computer_prefs_t dive_computer4;
+	bool        sync_dive_computer_time;
```

Comment:
that looks like something odd with whitespace

Reply:
It's aligned with the indentation of most of the rest of the `struct` members, like https://github.com/subsurface/subsurface/pull/3857/files#diff-18ce2a282c1f6acc704513f0cc9efa1b7d10cc7dd186305c6ea7368c433992c7L91.
I personally don't like aligned lists in code as they age very badly (see this discussion) and complicate maintenance, so happy to change this to single space.